index.html

```diff
@@ -24,11 +24,26 @@
                     </div>
                 </nav>
             </div>
+
+            <div class="desktop-header">
+              <div class="logo">
+                <a href=""><img src="My Logo.png" alt="" class="logo-img" /></a> 
+             </div>
+             <nav>
```

Comment:
- [ ] I kindly suggest that you ensure that the inside spacing on your left and right of your nav bar is the same as the Figma design.

|   |  |
| ------------- | ------------- |
| Your Design  | <img width="909" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/d8c58f73-13df-45d2-96a2-9c6b99cd7e0f"> |
| Figma Design  | <img width="553" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/247d1a5e-e5bc-4904-accb-836b114508af"> |